docs/guides/filesystem-usage.md

```diff
@@ -66,7 +67,7 @@ fs.put("dir", "my-repo/my-ref/dir", recursive=True)
     fs.put_file("my-repo/my-ref/file.txt", "file.txt", use_blockstore=True)
     ```
 
-    Direct lakeFS blockstore uploads require the installation of the corresponding `fsspec` file system implementation through `pip`.
```

Comment:
Quick Question: What is our rationale as to whether to put a tool in the backticks (``) or not?

Reply:
Rule of thumb was that whenever the name in the sentence refers to the product/tool itself as a whole, we don't put it in backticks.

When it specifically refers to a package name (as in the next sentence here), or a command to be run, we use backticks.

Here, `pip` kinda slipped through the cracks, but I think it could go either way, since it alludes to the command to be invoked.